README.md

```diff
@@ -2,59 +2,52 @@
-To stop the app run `docker-compose down`.
-
-Settings for the app can be changed in `.env`.
-Options for `.env`:
-- BACKEND_WORKERS - number of [workers used by gunicorn](https://docs.gunicorn.org/en/latest/run.html#commonly-used-arguments)
-- ANDROMEDA_RGB_SATELLITE_URL - URL pointing to a CSV file for joining RGB data
-
-### Certbot Setup on AWS
-Certificates can be installed to the EC2 VM using Certbot with Nginx on pip following [EFF instructions](https://certbot.eff.org/instructions?ws=nginx&os=pip).
+(optional) If you want to use or test a different branch of the Andromeda repository, checkout the desired branch.
+```bash
+cd Andromeda/Andromeda
+git checkout <branch>
+cd ..
+```
+2. Build the Docker image.
 ```bash
-sudo python3 -m venv /opt/certbot/
-sudo /opt/certbot/bin/pip install --upgrade pip
-sudo /opt/certbot/bin/pip install certbot certbot-nginx
-sudo ln -s /opt/certbot/bin/certbot /usr/bin/certbot
+docker image build -t andromeda .
 ```
-And to install Nginx on the Amazon Linux 2023 (following [instructions](https://awswithatiq.com/how-to-install-nginx-in-amazon-linux-2023/)) and run Certbot:
+3. Run the Docker image.
 ```bash
-docker-compose down
-sudo dnf update -y
-sudo dnf install nginx -y
-sudo systemctl start nginx
-sudo certbot certonly --nginx
+docker container run -p 7860:7860 andromeda
 ```
-Following prompted agreements, registration, and specifying andromeda.imageomics.org, the certificates are installed to:
 
-Certificate: `/etc/letsencrypt/live/andromeda.imageomics.org/fullchain.pem`
+While Hugging Face handles SSL encryption automatically, other deployment environments require SSL to be set up manually. One common approach is using a reverse proxy like Nginx. Below are some resources and tutorials that provide guidance on setting up SSL with Nginx in a Docker environment:
 
-Key: `/etc/letsencrypt/live/andromeda.imageomics.org/privkey.pem`
+- [Configure HTTPS for an Nginx Docker Container (Stackify)](https://stackify.com/configure-nginx-docker-container-https)​
+- [Setup SSL with Docker, NGINX and Lets Encrypt (Programonaut)​](https://www.programonaut.com/setup-ssl-with-docker-nginx-and-lets-encrypt)
+- [Set up SSL on Docker, Nginx Container and Let's Encrypt (SSLWiki)]​(https://sslwiki.org/docker-nginx-letsencrypt)
+- [Install TLS/SSL on Docker Nginx Container With Let’s Encrypt (Dev.to)](https://dev.to/techworldzone/how-to-install-tls-ssl-on-docker-nginx-container-with-let-s-encrypt-5a3n)
+- [NGINX Docker with SSL Encryption (Self-signed) by Mike Polinowski​](https://mpolinowski.github.io/nginx-docker-ssl)
 
-Finally, shut down Nginx on the system to avoid interference with the container:
-```bash
-sudo systemctl stop nginx
-docker-compose up -d
-```
+These tutorials cover a range of setups including self-signed certificates, Let's Encrypt, and Certbot. Choose the one (or more) that aligns well with your deployment scenario.
+
+Settings for the app can be changed in `.env`.
```

Comment:
I don't think the .env file will have any effect without running docker with `--env-file .env`. https://docs.docker.com/engine/reference/commandline/run/#env

Reply:
Should we keep just the `ANDROMEDA_RGB_SATELLITE_URL` in the `example.env` to keep the 'RGB Satellite Data' checkbox working?

I assume `BACKEND_WORKERS` is no longer relevant.

Reply:
### ANDROMEDA_RGB_SATELLITE_URL
I don't think HF is using the `.env` file when running the app. The default URL already points to the QUEST CSV:
https://github.com/Imageomics/Andromeda/blob/c5e6cd6e49a4b8b26eb529f56a283702fe78aba4/andromeda/satellitedata.py#L30
The `url_default` is used when `ANDROMEDA_RGB_SATELLITE_URL` is unset, which is the case in HF.
If someone was going to run Andromeda locally the example.env might be helpful.

### BACKEND_WORKERS
`BACKEND_WORKERS` has no effect with the current Dockerfile because we are running the app like so:
https://github.com/Imageomics/Andromeda/blob/77baf4e5a328283f7e34c86be14ed849bc581569/Dockerfile#L24

Flask recommends running gunicorn instead of running the app directly.
Could we change the Dockerfile to use the CMD from the deleted Dockerfile?
https://github.com/Imageomics/Andromeda/blob/c5e6cd6e49a4b8b26eb529f56a283702fe78aba4/andromeda/Dockerfile#L10
That way we could set `BACKEND_WORKERS` and hopefully accommodate more Andromeda users at the same time.